Function.cs

```diff
@@ -17,7 +17,10 @@ public class Function
 
         private readonly string[] coolColors =
         {
-            "Air Force Blue", 
+            "Air Force Blue",
+            "Antique bronze",
+            "Android green",
```

Comment:
Actually, Android Green and Antique Brass have already been added by others. We'll take your Antique Bronze, though!
```suggestion

```